app/services/rbac_api.rb

```diff
@@ -9,12 +9,16 @@ def initialize(b64_identity)
   end
 
   def check_user
-    body = JSON.parse(access_check_response.body)
-    body['data'].find do |access_check|
-      return true if access_check['permission'] == 'compliance:*:*'
+    begin
+      body = JSON.parse(access_check_response.body)
+      body['data'].find do |access_check|
+        return true if access_check['permission'] == 'compliance:*:*'
+      end
+    rescue Faraday::ClientError => e
+      Rails.logger.info("#{e.message} #{e.response}")
     end
-  rescue Faraday::ClientError => e
-    Rails.logger.info("#{e.message} #{e.response}")
+
+    false
```

Comment:
Best to use a real begin/rescue/end block instead of the very weirdly behaving in-method rescue/ensure. This makes it super clear what happens if everything falls through.